llvm/test/tools/llvm-objdump/XCOFF/file-headers.test

```diff
@@ -0,0 +1,21 @@
+# Test the `--file-headers` option of llvm-objdump for XCOFF object files.  
+# RUN: yaml2obj %s -o %t1
+# RUN: llvm-objdump --file-headers %t1 | FileCheck %s --check-prefix=CHECK32
+# RUN: yaml2obj %s -DMAGIC=0x1F7 -DFLAG64=0x2 -o %t2
+# RUN: llvm-objdump --file-headers  %t2 | FileCheck %s --check-prefix=CHECK64
```

Comment:
I suggest adding `--match-full-lines` to both `FileCheck` calls. Otherwise, there could be trailing output on the line that won't be picked up (for example, the 32-bit case will match both `start address: 0x00001111` and `start address: 0x0000111122223333` without it).

Reply:
thanks